spec/credit_card_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe EasyPost::CreditCard, :authenticate_prod do
+  describe '.fund' do
+    xit 'fund a credit card' do
+      # Skipping due to the lack of an available real credit card in tests
+      credit_card = described_class.fund('100', 'primary')
```

Comment:
Let's set the fund amount to something much smaller for the reasons mentioned in the other test suite. 

After talking with support, `20` is actually the minimum so let's use that here.

Reply:
20 sounds good